test/spmd/test_spmd_graph_dump.py

```diff
@@ -18,7 +18,7 @@ class BasicShardingTest(test_xla_sharding_base.XlaShardingTest):
 
   @classmethod
   def setUpClass(cls):
-    os.environ["XLA_USE_SPMD"] = "1"
+    xr.set_use_spmd()
```

Comment:
How about just `xr.use_spmd()`?

Reply:
lol, sounds good -- cc @JackCaoG wdyt?

Reply:
yea, sgtm

Reply:
I guess we never meant to provide a option to do `xr.use_spmd(False)` right? It should be a one time thing

Reply:
@JackCaoG true, right now (the first step) it's a drop-in replacement to the flag.